python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -858,32 +867,32 @@ def plot_stationary_position_error(self, truth_lla_deg):
         @param truth_lla_deg The truth LLA location (in degrees/meters).
         """
         truth_ecef_m = np.array(geodetic2ecef(*truth_lla_deg, deg=True))
-        self._plot_pose_displacement(title='Position Error', center_ecef_m=truth_ecef_m)
+        return self._plot_pose_displacement(title='Position Error', center_ecef_m=truth_ecef_m)
 
     def plot_pose_displacement(self):
         """!
         @brief Generate a topocentric (top-down) plot of position displacement, as well as plot of displacement over
                time.
         """
-        self._plot_pose_displacement()
+        return self._plot_pose_displacement()
```

Comment:
Two things:
1. What's the reason for returning this?
2. Should this be documented?

Reply:
In response to (1): looking around again, I guess this is consistent with the purpose of this PR.

Reply:
Yep. This isn't really useful internal to this file, but it's handy in our specific use case outside of this file where you want to use the `Analyzer` class instance to do the displacement calculation for you and then do something with that.